lib/hawkular/alerts/alerts_api.rb

```diff
@@ -210,6 +210,22 @@ def list_events(*criteria)
       query = generate_query_params(*criteria)
       http_get('/events' + query).map { |e| Event.new(e) }
     end
+
+    # Inject an event into Hawkular-alerts
+    # @param [String] id Id of the event must be unique
+    # @param [String] category Event category for further distinction
+    # @param [String] text Some text to the user
+    # @param [Hash<String,Object>] context Additional context information.
+    def create_event(id, category, text, context = {})
+      event = {}
+      event['id'] = id
+      event['ctime'] = Time.now.to_i * 1000
+      event['category'] = category
+      event['text'] = text
+      event['context'] = context unless context.nil?
```

Comment:
@abonas if someone passes in 'nil' explicitly
@jshaughn I could for sure just expect a hash or a `Event` object, but as the `Event` and others are `ostruct`, there is virtually no guidance in the gem-api on what is expected as input, but yes expecting some explicitly and then using the remainder from the hash would certainly work.